docs/index.md

```diff
@@ -0,0 +1,14 @@
+The Arduino Firmware Uploader is a tool made to update the firmware and/or add SSL certificates for any arduino board
+equipped with WINC or NINA module.
```

Comment:
If change is made in response to my comment to the equivalent text in README.md, do the same here.